package.json

```diff
@@ -53,28 +53,32 @@
             "<rootDir>/node_modules/"
         ]
     },
+    "resolutions": {
+        "@types/react": "17.0.14",
+        "@types/react-dom": "17.0.14"
+    },
```

Comment:
@nina992 Can you explain why these are necessary? I think they came up in a solution you found for a problem you were having in your local environment, but it would be better if you fixed your local environment rather than adding to our codebase (unless absolutely necessary).